src/libraries/System.Utf8String.Experimental/ref/System.Utf8String.Experimental.csproj

```diff
@@ -1,15 +1,36 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- disable warnings about obsolete APIs -->
     <NoWarn>$(NoWarn);0809;0618</NoWarn>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.0;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Do you n eed the netcoreapp3.0 configuration as well? I see why you need the netsatndard2.1 config, but I'm not seeing why netcoreapp3.0 is required.

Reply:
`netcoreapp3.0` is necessary because `System.Text.Rune` is not in `netstandard2.1` but we shipped it in `netcoreapp3.0`. So if someone used this package on `netcoreapp3.x`, they would get a "duplicate type" build error for `System.Text.Rune`. So we need to build for `netcoreapp3.0` and TypeForward `Rune`.